pkg/3scale/amp/component/system_sphinx.go

```diff
@@ -0,0 +1,170 @@
+package component
+
+import (
+	"fmt"
+
+	appsv1 "github.com/openshift/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	SystemSphinxDeploymentName = "system-sphinx"
+	SystemSphinxPVCName        = "system-sphinx"
+	SystemSphinxServiceName    = "system-sphinx"
+	SystemSphinxDBVolumeName   = "system-sphinx-databasex"
```

Comment:
Was the `x` at the end in `databasex` intentional?

Reply:
It would be cool, but, it is a typo :) fixing it.